tendrl/monitoring_integration/grafana/alert_dashboard.py

```diff
@@ -362,7 +362,7 @@ def add_gluster_resource_panel(
                 host_name = resource_name.split("|", 1)[1].split(
                     ":", 1)[0].replace(".", "_")
                 brick_name = resource_name.split("|", 1)[1].split(
-                    ":", 1)[1].replace("/", "|")
+                    ":", 1)[1]
```

Comment:
it just for alert panel title, we will give slash (/) as it is

Reply:
Ack. Hope there is no regression due to this.

Reply:
i verified this scenario, it works fine